isso/db/__init__.py

```diff
@@ -121,3 +121,39 @@ def first(rv):
 
                 con.execute('PRAGMA user_version = 3')
                 logger.info("%i rows changed", con.total_changes)
+
+        # "text" field in "comments" became NOT NULL
+        if self.version == 3:
+            with sqlite3.connect(self.path) as con:
+                con.execute("BEGIN TRANSACTION")
+                con.execute("UPDATE comments SET text = '' WHERE text IS NULL")
+                logger.info("%i rows changed", con.total_changes)
```

Comment:
I know this is a copy-paste from the other migration steps, but can we make this more informative? E.g. `Migrating DB version 3 to 4 by setting empty comments.text to '', %i rows changed`